src/Routes.js

```diff
@@ -24,6 +27,10 @@ const renderRoutes = () => (
         <Route path="/profile" exact component={ProfilePage} />
       </SwipeableRoutes>
     </MainLayout>
+    <DocsLayout>
```

Comment:
> The /privacy and /terms-and-conditions and now rendering beneath home-page.

Add it in the switch instead.

I have added switch in the latest code of main branch please see it.